packages/mendel-development-loader/loader.js

```diff
@@ -13,11 +15,45 @@ function MendelLoaderDev(existingVariations, config, parentModule) {
     this._existingVariations = existingVariations;
     this._config = config;
     this._parentModule = parentModule || module.parent;
+    this._ssrReady = isValidDir(config.serveroutdir);
 }
 
 MendelLoaderDev.prototype.resolver = function(variations) {
+    var config = this._config;
+
+    if (!this.isSsrReady()) {
+        console.warn('Warning: Mendel server output dir found. Falling back to node require.');
+        return {
+            require: this._fallback_require.bind(this),
+            resolve: this._fallback_resolve.bind(this),
+        };
+    }
+
     var dirs = resolveVariations(this._existingVariations, variations);
-    return new MendelResolverDev(this._parentModule, dirs, this._config);
-}
+    return new MendelResolverDev(this._parentModule, dirs, config);
+};
+
+MendelLoaderDev.prototype.isSsrReady = function() {
+    return this._ssrReady;
+};
+
+MendelLoaderDev.prototype._fallback_resolve = function (mod) {
+    var config = this._config;
+    return path.resolve(config.basedir, config.basetree, mod);
```

Comment:
I see how this might help, but I would prefer to render nothing than rendering the source of the base tree.

If babel register is not being used, it would just throw. Also, if the user ins on and experiment on the client bundle, there will be mismatch of markup and rehydration would fail.

Can you help me better understand how this is helping?